src/main/java/uk/ac/cam/cl/dtg/segue/etl/ContentIndexer.java

```diff
@@ -605,11 +605,8 @@ private synchronized void buildElasticSearchIndex(final String sha,
                                                       final Map<String, String> allUnits,
                                                       final Map<String, String> publishedUnits,
                                                       final Map<Content, List<String>> indexProblemCache) {
-        if (es.hasIndex(sha, CONTENT_INDEX_TYPE.CONTENT.toString())) {
-            log.info("Deleting existing indexes for version " + sha);
```

Comment:
Could this log line not have stayed?

Reply:
I moved it into `expungeAnyContentTypeIndicesRelatedToVersion(...)` so that it is logged wherever it id called